.env.example

```diff
@@ -74,9 +74,9 @@ POSTGRES_USE_PGVECTOR=true
 # POSTGRES_SSL_DIR='/var/lib/postgresql'
 REDIS_PASSWORD=''
 REDIS_URL='redis://localhost:6379'
-# REDIS_TLS_CERT_FILE=./certs/redis.crt
-# REDIS_TLS_KEY_FILE=./certs/redis.key
-# REDIS_TLS_CA_FILE=./certs/redisCA.crt
+# REDIS_TLS_CERT_FILE=./run-it/development/redis/certs/redis.crt
```

Comment:
nit: prefer camelCase to kebab-case for naming convention. Instead of `run-it` perhaps another name like `run, deploy, infra, devOps`? same for `singleTenantHost`.


nit: why have both a `docker` and `run-it` directory? Both are concerned with how the thing is built. `./run-it/single-tenant-host` has a docker-copmose file init. Could we combine those 2 directories? Into a single e.g. `deploy`?

Reply:
Naming: I do not really care, we can just switch to run instead of run-it. We have more folders using - and I prefer that too, but I do not have a strong opinion on it 😸 we have parabol-ubi (which had inside docker-build and docker-host-st), we have packages/gql-executor and integration-tests

About the different folders:
- Docker is everything that is to build docker images. As you can see, not only Parabol.
- We could host other ways of running the application without Docker inside of it.
- We should not combine single-tenant-host and development. Those are completely different. 
- Single-tenant isn't a meant for development. Single-Tenant is what we use for PPMIs and what we would provide to Self-Hosted clients that would like to see how to make the app run in the simplest way. 
- I would like to keep our development folder so you guys can change it and use it. In the development docker-compose we do not run the application.

Reply:
`run-it` moved to `run`. Let's keep the rest as it is for now and re-evaluate in the future if there is any tension